hack/deploy_only_cr.sh

```diff
@@ -17,16 +17,35 @@ fi
 function cleanup() {
     rv=$?
     if [ "x$rv" != "x0" ]; then
-        echo "Error during upgrade: exit status: $rv"
+        echo "Error during HCO CR deployment: exit status: $rv"
         make dump-state
-        echo "*** Upgrade test failed ***"
+        echo "*** HCO CR deployment failed ***"
     fi
     exit $rv
 }
 
 trap "cleanup" INT TERM EXIT
 
+if [[ -n "${KVM_EMULATION}" ]]; then
+  SUBSCRIPTION_NAME=$(oc get subscription -n "${HCO_NAMESPACE}" -o name)
+  # cut the type prefix, e.g. subscription.operators.coreos.com/kubevirt-hyperconverged => kubevirt-hyperconverged
+  SUBSCRIPTION_NAME=${SUBSCRIPTION_NAME/*\//}
+
+  TMP_DIR=$(mktemp -d)
+  cat > "${TMP_DIR}/subscription-patch.yaml" << EOF
+spec:
+  config:
+    selector:
+      matchLabels:
+        name: hyperconverged-cluster-operator
+    env:
+    - name: 'KVM_EMULATION'
+      value: "${KVM_EMULATION}"
+EOF
+
+  ${CMD} patch -n "${HCO_NAMESPACE}" Subscription "${SUBSCRIPTION_NAME}" --patch="$(cat "${TMP_DIR}/subscription-patch.yaml")" --type=merge
+fi
+
```

Comment:
Patching the subscription to inject KVM_EMULATION variable will cause all the deployments to be patched and this will cause all the operators pods to be restarted; maybe it's safer to wait for HCO pod to be ready again before creating the CR.

Reply:
done